index.js

```diff
@@ -504,6 +504,10 @@ function launchApp(argo, startExternalAdapterServer) {
             configObject: { licenseKey }
         } = configuration;
 
+        if (argo['user-app-config-args']) {
+            configObject['startup_app'].url = `${configObject['startup_app'].url}?${argo['user-app-config-args']}`;
```

Comment:
I think we need to accommodate original `url` possibly already having a querystring by appending with `&` rather than `?`, something like :
```js
var options = configObject['startup_app'];
configObject['startup_app'].url += options.url.indexOf('?') < 0 ? '?' : '&';
options.url += argo['user-app-config-args'];
```
Because multiple `?` would be unexpected.
This does raise the question however as to whether RVM `$$` params should _overwrite_ existing url params or should just be appended as duplicate keys (permitted in urls).